File: Mundial.Infra/Model/Address.cs
using System;

namespace Mundial.Infra.Model
{
    public class Address: MundialModel
    {
        public string City {get; set;}

        public string Street {get; set;}

        public string Neighborhood {get; set;}

        public int HouseNumber {get; set;}

        public string Complement {get; set;}

        public string CEP {get; set;}
    }
}

File: Client/mundial-client/src/app/models/address.ts
import { MundialModel } from './mundial-model';

export class Address extends MundialModel {
  public City: String;

  public Street: String;

  public Neighborhood: String;

  public HouseNumber: Number;

  public Complement: String;

  public CEP: String;


	constructor(City: String, Street: String,  Neighborhood: String, HouseNumber: Number,
              Complement: String,  CEP: String, CreationDate: Date)
  {

    super(CreationDate);

    this.City = City;
    this.Street = Street;
    this.CEP = CEP;
    this.Neighborhood = Neighborhood;
    this.HouseNumber = HouseNumber;
    this.Complement = Complement;
  }

}
